velox/dwio/common/Options.h

```diff
@@ -491,6 +491,11 @@ class ReaderOptions : public io::ReaderOptions {
     return *this;
   }
 
+  ReaderOptions& setUseParquetColumnNamesForColumnMapping(bool flag) {
+    useParquetColumnNamesForColumnMapping_ = flag;
```

Comment:
Just reuse `useColumnNamesForColumnMapping_`, no need to create one specifically for parquet